include/deal.II/grid/manifold_lib.h

```diff
@@ -25,37 +25,37 @@
 DEAL_II_NAMESPACE_OPEN
 
 /**
- * Manifold description for a spherical space coordinate system.
+ * Manifold description for a polar coordinate system.
  *
  * You can use this Manifold object to describe any sphere, circle,
- * hypersphere or hyperdisc in two or three dimensions, both as a co-dimension
- * one manifold descriptor or as co-dimension zero manifold descriptor.
+ * hypersphere or hyperdisc in two or three dimensions, both as a
+ * co-dimension one manifold descriptor or as co-dimension zero
+ * manifold descriptor, provided that the north and south poles (in
+ * three dimensions) are excluded from the Manifold (as they are
+ * singular).
  *
  * The two template arguments match the meaning of the two template arguments
  * in Triangulation<dim, spacedim>, however this Manifold can be used to
  * describe both thin and thick objects, and the behavior is identical when
- * dim <= spacedim, i.e., the functionality of SphericalManifold<2,3> is
- * identical to SphericalManifold<3,3>.
- *
- * The two dimensional implementation of this class works by transforming
- * points to spherical coordinates, taking the average in that coordinate
- * system, and then transforming back the point to Cartesian coordinates. For
- * the three dimensional case, we use a simpler approach: we take the average
- * of the norm of the points, and use this value to shift the average point
- * along the radial direction. In order for this manifold to work correctly,
- * it cannot be attached to cells containing the center of the coordinate
- * system. This point is a singular point of the coordinate transformation,
- * and there taking averages does not make any sense.
+ * dim <= spacedim, i.e., the functionality of PolarManifold<2,3> is
+ * identical to PolarManifold<3,3>.
  *
- * This class is used in step-1 and step-2 to describe the boundaries of
- * circles. Its use is also discussed in the results section of step-6.
+ * This class works by transforming points to polar coordinates (in
+ * both two and three dimensions), taking the average in that
+ * coordinate system, and then transforming back the point to
+ * Cartesian coordinates. In order for this manifold to work
+ * correctly, it cannot be attached to cells containing the center of
+ * the coordinate system or the north and south poles in three
+ * dimensions. These points are singular points of the coordinate
```

Comment:
Actually no. In 2d we have periodicity, but no singularity. In other words, it is possible to write a single transformation with Jacobian different from zero everywhere (in 2D) (provided we allow for periodicity).